onnxruntime/core/providers/rocm/tunable/gemm_rocblas.h

```diff
@@ -169,16 +190,18 @@ class RocBlasGemmTunableOp : public TunableOp<GemmParams<T>> {
  private:
   std::vector<int> GetSolutions(const GemmParams<T>* params) {
     int num_solutions = 0;
+    auto h_a = DoCastForHalfOrBfloat16(params->alpha);
```

Comment:
Minor, here and elsewhere, consider inlining the definitions of `h_a` and `h_b`. I feel that it is more readable that way.